examples/mega65/drums.license

```diff
@@ -0,0 +1,4 @@
+Source:      https://commons.wikimedia.org/wiki/File:Patró_de_bateria.wav
+Attribution: Escola Superior de Música de Catalunya (ESMUC)
+License:     CC BY-SA 3.0 <https://creativecommons.org/licenses/by-sa/3.0>, via Wikimedia Commons
```

Comment:
This will cause problems. Share-Alike's terms require Adaptations to be licensed under the same license, and the example above is closer to an adaption or derivative work of the drum sample than it is a simple "collection". We'd need to find something else to use either with a compatible license to Apache 2 or in the public domain.